sdk/core/src/http.rs

```diff
@@ -0,0 +1,44 @@
+pub struct Request {
+    inner: http::Request<bytes::Bytes>,
+}
```

Comment:
As discussed yesterday in our meeting, this will be an enum accepting either a stream (AsyncRead+AsyncSeek) or a `bytes::Bytes`. We should merge this as it is now and build upon it.